assignments/assignment0.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Assignment 0 - GIT Instructions
+nav_exclude: true
+---
+
+# GIT Instructions
+
+## 1. Set Up Your GitHub Repository:
+- Create a new repository on GitHub named `username.github.io`, where `username` is your GitHub username.
+- This repository will host your personal website.
+
+## 2. Design Your Website:
+- Create a simple HTML/CSS website. You can use any website template you like or build your own from scratch.
+- Ensure that your website includes the following sections:
+  - **About Me**: A brief introduction about yourself.
+  - **Work Experience**: If any, highlight your work experience.
+  - **Projects**: A section showcasing some of the projects you've worked on.
+  - **Contact**: Information on how to reach you, like email, LinkedIn, etc.
+  - Each section should be a separate page. Don't include multiple sections in same page, create different dedicated pages for different sections.
```

Comment:
More details needed on the url of each. i.e. be explicit that the urls should look like username.github.io/assignments